magento/pwa-studio
# Venia v2 Cart Page

## Description

Venia v2 separates out the Cart into its own page. This PR lays out the basic structure of the page. Future PRs will populate the page with its actual content / components.

## Related Issue





Closes [PWA-130](https://jira.corp.magento.com/browse/PWA-130).

## Acceptance 



### Verification Stakeholders


@jimbo 

### Specification



### Verification Steps


1. `yarn watch:venia`
1. Navigate to Venia in your browser
1. Navigate to `<your dev site>/cart`
1. Verify that the new Cart page skeleton appears
1. Verify that if you are not signed in, a "Sign In" button appears
1. Verify that clicking the "Sign In" button opens the navigation drawer.
1. Verify that if you are already signed in, the "Sign In" button does not appear
1. Switch between Desktop and Mobile (< `640px`) views

On Mobile:
1. Verify that the "Recently Viewed" container is not present
1. Verify that the page is a single column layout
1. Verify that the "Order Summary" container appears at the bottom

On Desktop:
1. Verify that the "Recently Viewed" container is present
1. Verify that the page is split into two columns
1. Scroll and verify that the "Order Summary" container "sticks" to the side / stays in view. Note that it is expected that it will disappear once you scroll past all of the content.

## Screenshots / Screen Captures (if appropriate)

**Mobile**

![Screen Shot 2020-01-07 at 2 44 03 PM](https://user-images.githubusercontent.com/13182778/71927974-3d18a900-315c-11ea-9f02-a7bc80d4d4c3.png)

**Desktop**

![Screen Shot 2020-01-07 at 2 44 22 PM](https://user-images.githubusercontent.com/13182778/71927982-41dd5d00-315c-11ea-8a56-7e633664155a.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
